assets/stylesheets/sections/_checkout.scss

```diff
@@ -131,36 +131,6 @@
 	.payment-box__content {
 		min-height: 140px;
 		margin-top: 10px;
-
-		.cart-coupon {
```

Comment:
Yup, these were needed here because for the test we've added the coupon to the different payment boxes :+1: